src/ROUTES.ts

```diff
@@ -615,6 +615,14 @@ const ROUTES = {
         route: 'r/:reportID/transaction/:transactionID/receipt',
         getRoute: (reportID: string, transactionID: string) => `r/${reportID}/transaction/${transactionID}/receipt` as const,
     },
+    POLICY_ACCOUNTING_QUICKBOOKSONLINE_IMPORT: {
+        route: 'settings/workspaces/:policyID/accounting/quickbooks-online/import',
+        getRoute: (policyID: string) => `workspace/${policyID}/connections/quickbooks-online/import` as const,
+    },
+    POLICY_ACCOUNTING_QUICKBOOKSONLINE_OPTION: {
+        route: 'workspace/:policyID/accounting/quickbooks-online/import/:option',
+        getRoute: (policyID: string, option: string) => `workspace/${policyID}/connections/quickbooks-online/import/${option}` as const,
```

Comment:
if we're going to make each import screen dynamic route, then we should create a TypeScript type that is a union of possible route postfixes (ex. `chart`, `class`, ...)

Reply:
actually, I don't think we should do this (i.e. making the last part of the route dynamic) 😄 

if we do this, we have to make one giant screen that implements all the possible import settings views.

or did you write this code as a place holder?

Reply:
yup - it's just a getRoute - each specific import item will have own page